### PR TITLE
Back end

### PR DESCRIPTION
**Pull request by BlackOrangee**

### CODE REVIEW

**Review by MaxTrgb — APPROVED** [review]
1

File: Back-end/Controller/UserController.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    {
+        private UserService _userService = new UserService();
+        private PostService _postService = new PostService();
+        private CommentService _commentService = new CommentService();
+
+        private readonly string _basePath = "/api/user";
+
+        public UserController()
+        {
+            //Get(_basePath + "/", _ => "{\"message\": \"Hello World!\"}");
```
Comment (XardasCode) [inline Back-end/Controller/UserController.cs]: Remove?

File: Back-end/Repository/PostRepository.cs [hunk 2]
```diff
@@ -0,0 +1,171 @@
+﻿using DENMAP_SERVER.Entity;
+using MySql.Data.MySqlClient;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Comment (XardasCode) [inline Back-end/Repository/PostRepository.cs]: It not used here, please clean using here and below

File: Back-end/Service/PostService.cs [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DENMAP_SERVER.Service
+{
+    internal class PostService
+    {
+        string connectionString = "Server=34.116.253.154;Port=3306;Database=chat_database;Uid=app_user;Pwd=&X9fT#7vYqZ$4LpR;";
```
Comment (XardasCode) [inline Back-end/Service/PostService.cs:14]: Push this only if this repository is private

File: Back-end/Service/UserService.cs [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DENMAP_SERVER.Service
+{
+    internal class UserService
+    {
+        string connectionString = "Server=34.116.253.154;Port=3306;Database=chat_database;Uid=app_user;Pwd=&X9fT#7vYqZ$4LpR;";
```
Comment (XardasCode) [inline Back-end/Service/UserService.cs:14]: ditto

File: Back-end/Service/UserService.cs [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+        }
+
+        public int loginUser(string name, string password)
+        {
+            User user = null;
+
+            using (MySqlConnection connection = new MySqlConnection(connectionString))
+            {
+                connection.Open();
+                System.Console.WriteLine("User service, starting login");
```
Comment (XardasCode) [inline Back-end/Service/UserService.cs]: Create real logger to this project that will log also to file(in the future commits)

File: Back-end/obj/Debug/net8.0/DENMAP_SERVER.csproj.FileListAbsolute.txt [hunk 6]
```diff
@@ -20,3 +20,12 @@ G:\cw2\DENMAP_SERVER\obj\Debug\net8.0\refint\DENMAP_SERVER.dll
 G:\cw2\DENMAP_SERVER\obj\Debug\net8.0\DENMAP_SERVER.pdb
 G:\cw2\DENMAP_SERVER\obj\Debug\net8.0\DENMAP_SERVER.genruntimeconfig.cache
 G:\cw2\DENMAP_SERVER\obj\Debug\net8.0\ref\DENMAP_SERVER.dll
+C:\Users\CC\Documents\git\my-app\Back-end\obj\Debug\net8.0\DENMAP_SERVER.csproj.AssemblyReference.cache
```
Comment (XardasCode) [inline Back-end/obj/Debug/net8.0/DENMAP_SERVER.csproj.FileListAbsolute.txt]: For such files must be created gitignore

File: Back-end/obj/Debug/net8.0/DENMAP_SERVER.sourcelink.json [hunk 7]
```diff
@@ -0,0 +1 @@
+{"documents":{"C:\\Users\\CC\\Documents\\git\\my-app\\*":"https://raw.githubusercontent.com/MaxTrgb/my-app/23be836973486d38b7390193099528996fde818c/*"}}
```
Comment (XardasCode) [inline Back-end/obj/Debug/net8.0/DENMAP_SERVER.sourcelink.json]: ditto

File: Back-end/obj/Debug/net8.0/DENMAP_SERVER.AssemblyInfo.cs [hunk 8]
```diff
@@ -14,7 +14,9 @@
 [assembly: System.Reflection.AssemblyCompanyAttribute("DENMAP_SERVER")]
 [assembly: System.Reflection.AssemblyConfigurationAttribute("Debug")]
 [assembly: System.Reflection.AssemblyFileVersionAttribute("1.0.0.0")]
```
Comment (XardasCode) [inline Back-end/obj/Debug/net8.0/DENMAP_SERVER.AssemblyInfo.cs:16]: create gitignore

File: Back-end/obj/Debug/net8.0/DENMAP_SERVER.AssemblyInfoInputs.cache [hunk 9]
```diff
@@ -1 +1,3 @@
+
 2dfee7b84fdd521fe33945bde9aecc8206248befc5e0ff4926a3561ccc7e834c
```
Comment (XardasCode) [inline Back-end/obj/Debug/net8.0/DENMAP_SERVER.AssemblyInfoInputs.cache]: ditto

**Pull request merged**